geometry/geometry_config.py

```diff
@@ -608,19 +524,20 @@
         c.tauMudet.PillarX = 40*u.cm
         c.tauMudet.PillarZ = 50*u.cm
         c.tauMudet.PillarY = 10*u.m - c.cave.floorHeightMuonShield - c.tauMudet.Ytot/2 - 0.1*u.mm
-    if nuTauTargetDesign<4:
-     c.tauMudet.XGas =  c.tauMudet.XRpc
-     c.tauMudet.YGas =  c.tauMudet.YRpc
-     c.tauMudet.ZGas = 1*u.mm
-     c.tauMudet.XStrip =  c.tauMudet.XRpc
-     c.tauMudet.YStrip =  c.tauMudet.YRpc
-     c.tauMudet.ZStrip = 0.02*u.mm
-     c.tauMudet.XPet =  c.tauMudet.XRpc
-     c.tauMudet.YPet =  c.tauMudet.YRpc
-     c.tauMudet.ZPet = 0.1*u.mm
-     c.tauMudet.XEle =  c.tauMudet.XRpc
-     c.tauMudet.YEle =  c.tauMudet.YRpc
-     c.tauMudet.ZEle = 1*u.mm
+
+        c.tauMudet.XGas =  c.tauMudet.XRpc
+        c.tauMudet.YGas =  c.tauMudet.YRpc
+        c.tauMudet.ZGas = 1*u.mm
+        c.tauMudet.XStrip =  c.tauMudet.XRpc
+        c.tauMudet.YStrip =  c.tauMudet.YRpc
+        c.tauMudet.ZStrip = 0.02*u.mm
+        c.tauMudet.XPet =  c.tauMudet.XRpc
+        c.tauMudet.YPet =  c.tauMudet.YRpc
+        c.tauMudet.ZPet = 0.1*u.mm
+        c.tauMudet.XEle =  c.tauMudet.XRpc
+        c.tauMudet.YEle =  c.tauMudet.YRpc
+        c.tauMudet.ZEle = 1*u.mm
+        c.EmuMagnet.zC = c.tauMudet.zMudetC - c.tauMudet.Ztot/2 - c.EmuMagnet.GapDown - c.EmuMagnet.Z/2
```

Comment:
A lot of these lines have two spaces after the equals sign. One space before and after each operator please.